ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Expected Behavior
+
+(Describe how things should work.)
+
+
+### Actual Behavior
+
+(Describe what went wrong.)
+
+
+### Steps to Reproduce the issue
+
+(Describe how someone can see a working example of the issue. Include what version of Ember.js/Ember CLI you are using, and any details regarding browser specific behaviors. Is there a feature flag that needs to be enabled to reproduce the issue?)
+
+
+### Attempts to Solve the issue
+
+(Describe what you have tried to solve this issue. Did you try an earlier release of Ember, or perhaps beta/canary release channels?)
+
+
+### Before submitting this issue, please review the following…
```

Comment:
Should add a checklist item for "Am I in the right repo?" is this an issue for the Guides? Website? etc.